report_generator.py

```
@@ -32,6 +32,36 @@ def _format_positive_timedelta(self, td):
+
+        # Calcula la diferencia en días con la ausencia anterior del mismo empleado
+        df_absences['dias_desde_anterior'] = df_absences.groupby('employee')['dia'].diff().dt.days
+
+        # Un nuevo episodio comienza si es la primera falta o si han pasado más de 1 día
+        # desde la falta anterior (es decir, no es consecutiva).
+        df_absences['nuevo_episodio'] = (df_absences['dias_desde_anterior'].isna()) | (df_absences['dias_desde_anterior'] > 1)
+
+        # Sumar los inicios de nuevos episodios por empleado
+        episode_counts = df_absences.groupby('employee')['nuevo_episodio'].sum().astype(int)
+
+        return episode_counts
```

Ruff (0.12.2)

Unnecessary assignment to `episode_counts` before `return` statement

Remove unnecessary assignment

(RET504)